templates/naemon_monitor/mimir-server.yaml

```diff
@@ -0,0 +1,47 @@
+# Do not use this configuration in production.
```

Comment:
Should the comment that this should not be used in production still be here? Do we need to tweak something in order to make it "production ready"?

Reply:
Copy paste error. That was from an example storing persistent data i /tmp